metron-docker/kafkazk/bin/wait-for-it.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
```

Comment:
For security reasons, Docker will not pull in any assets outside of the directory containing the Dockerfile.  You have to do that separately or provide a script.  

We do have a script that copies in Metron dependencies at metron-docker/install-metron.sh.  Should we move the wait-for-it.sh script up to metron-docker and add steps to copy to images in install-metron.sh?